components/user-mgt/org.wso2.carbon.identity.user.pre.update.profile.action/src/main/java/org/wso2/carbon/identity/user/pre/update/profile/action/internal/management/PreUpdateProfileActionDTOModelResolver.java

```diff
@@ -123,7 +133,8 @@ public ActionDTO resolveForUpdateOperation(ActionDTO updatingActionDTO, ActionDT
         Map<String, ActionPropertyForDAO> properties = new HashMap<>();
         // Action Properties updating operation is treated as a PUT in DAO layer. Therefore if no properties are updated
         // the existing properties should be sent to the DAO layer.
-        List<String> attributes = getResolvedUpdatingOrExistingAttributes(updatingActionDTO, existingActionDTO);
+        List<String> attributes = getResolvedUpdatingOrExistingAttributes(updatingActionDTO, existingActionDTO,
```

Comment:
```suggestion
        List<String> attributes = getResolvedUpdatingAttributes(updatingActionDTO, existingActionDTO,
```

Shall we rename this method ? We dont need to mention the existing part

Reply:
Fixed with 2bdf593bd8c6dbf23b83131eaffbb3d789352045